Controller/CoreController.php

```diff
@@ -129,10 +124,27 @@ public function searchAction(Request $request)
         }
 
         return $this->render($this->container->get('sonata.admin.pool')->getTemplate('search'), array(
-            'base_template' => $this->getBaseTemplate($request),
+            'base_template' => $this->getBaseTemplate(),
             'admin_pool'    => $this->container->get('sonata.admin.pool'),
             'query'         => $request->get('q'),
             'groups'        => $this->getAdminPool()->getDashboardGroups(),
         ));
     }
+
+    /**
+     * The purpose of this method it to be BC with symfony 2.
```

Comment:
Get the request object from the container.

This method is compatible with both Symfony 2.3 and Symfony 3.